tests/ffi/agent-flood/test.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+
+. ../common/prepare.sh
+
+export QM_HOST_REGISTRY_DIR="/var/qm/lib/containers/registry"
+export QM_REGISTRY_DIR="/var/lib/containers/registry"
+export NUMBER_OF_NODES="${NUMBER_OF_NODES:-2}"
+WAIT_BLUECHI_AGENT_CONNECT="${WAIT_BLUECHI_AGENT_CONNECT:-5}"
+
+setup_test_containers_in_qm() {
+
+    #Prepare quadlet files for testing containers
+    for ((i=1;i<=NUMBER_OF_NODES;i++)); do
+        info_message "setup_test_containers_in_qm(): prepare quadlet files for bluechi-tester-${i}.container"
+        cat >> "/etc/qm/containers/systemd/bluechi-tester-${i}.container" <<EOF
+[Unit]
+Description=bluechi-tester-X
+After=local-fs.target
+
+[Container]
+Image=dir:/var/lib/containers/registry/tools-ffi:latest
+Exec=/root/tests/FFI/bin/bluechi-tester --url="tcp:host=localhost,port=842" \
+     --nodename=bluechi-tester-X \
+     --numbersignals=11111111 \
```

Comment:
another question: can the numbersignals and signal be a parameter from maybe main.fmf (env variable maybe)?

Reply:
opened an issue for this one.